Assets/Scripts/UIStates/MainMenuState.cs

```diff
@@ -0,0 +1,7 @@
+public class MainMenuState : UIBaseState
+{
+    protected override void EnterState() => TimeScaleController.SetTimeScale(0);

```

Comment:
Zou je niet juist willen hebben dat de time scale hier 1 is?

Reply:
nee